src/repository/postgres/doctor.js

```diff
@@ -0,0 +1,83 @@
+export default class PGDoctor {
+  constructor(pool) {
+    this.pool = pool;
+  }
+
+  getAll = async () => {
+    const result = [];
+
+    const query = `
+      SELECT doctors.*, specializations.title 
+      FROM doctors INNER JOIN specializations 
+        ON doctors.specialization_id = specializations.id
+    `;
+
+    const doctors = await this.pool.query(query);
+
+    doctors.rows.forEach((doctor) => {
```

Comment:
.map method could remove some lines of code

Reply:
:white_check_mark: Done